src/fd.cpp

```diff
@@ -111,12 +128,46 @@ static int wait_thread(void *arg) {
     }
```

Comment:
Could use `std::fill` here

Reply:
Thanks! have amended in [1bafc3b](https://github.com/shikokuchuo/later/pull/1/commits/1bafc3beb9a4ad49d617a8ff229ce094dfa3d3e8).